custom/icds_reports/tasks.py

```diff
@@ -1676,3 +1677,18 @@ def _get_primary_data_for_cases(db, domain, day, case_type):
         type=case_type
     )
     return matching_cases.values_list('case_id', 'type', 'server_modified_on')
+
+
+#To on 6AM IST
+@periodic_task_on_envs(
+    settings.ICDS_ENVS,
+    run_every=crontab(minute=30, hour=12),
```

Comment:
```suggestion
    run_every=crontab(minute=30, hour=12),  # 6 AM IST
```

nit: could remove comment on top of function. I think its a bit nicer to keep a comment with the exact line it is for in this case

Reply:
:+1: